model/distro/distro_test.go

```diff
@@ -425,6 +425,7 @@ func TestGetResolvedPlannerSettings(t *testing.T) {
 	// Fallback to the SchedulerConfig.ExpectedRuntimeFactor as PlannerSettings.ExpectedRunTimeFactor is equal to 0.
 	assert.EqualValues(t, 7, resolved0.ExpectedRuntimeFactor)
 	assert.EqualValues(t, 20, resolved0.GenerateTaskFactor)
+	//nolint:testifylint // We expect it to be exactly 10.
```

Comment:
Out of curiosity, what does testifylint not like about this + the others? It seems like checking that some output value exactly matches the expected value would be a normal thing to do in a test.

Reply:
These ones are checking float. Testifylint suggests we use a (I didn't know it existed) `InEpsilon` assertion/require for floats to account for floating point number errors. I figured since our tests are passing, no reason to modify the behavior to check for a range.